mis_builder/migrations/16.0.1.0.1/post-migration.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    """Set the value of the analytic_domain field."""
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE mis_report_instance_period
+        SET analytic_domain = CONCAT('[("analytic_account_id", "=", ', analytic_account_id::VARCHAR, ')]')
+        WHERE analytic_account_id IS NOT NULL
+        """,  # noqa: E501
+    )
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE mis_report_instance
+        SET analytic_domain = CONCAT('[("analytic_account_id", "=", ', analytic_account_id::VARCHAR, ')]')
```

Comment:
Does this domain work with out of the box v16 accounting?

Reply:
This is applied over `account.move.line`, isn't it? Indeed, I don't find a field `analytic_account_id` on such model.

Reply:
You are right, I change it to `analytic_line_ids.account_id` to make it correct, ok?

Reply:
Well, I see no other option. Good for you, @sbidoul ?